amd_openvx/openvx/hipvx/filter_kernels.cpp

```diff
@@ -7302,4 +7302,4 @@ int HipExec_ScaleGaussianHalf_U8_U8_5x5(hipStream_t stream, vx_uint32 dstWidth,
                         dstWidthComp);
 
     return VX_SUCCESS;
-}
+}
```

Comment:
why this file is even touched in this PR? please sync with ToT for this file.

Reply:
Sure. I am updating the PR will do it.